build-tools/package.json

```diff
@@ -30,8 +31,8 @@
 		"format": "npm run prettier:fix",
 		"preinstall": "node ../scripts/only-pnpm.cjs",
 		"install:commitlint": "npm install --global @commitlint/config-conventional",
-		"lint": "lerna run lint --no-sort --stream",
-		"lint:fix": "lerna run lint:fix --no-sort --stream",
+		"lint": "npm run prettier && npm run ci:lint",
+		"lint:fix": "npm run prettier:fix && lerna run lint:fix --no-sort --stream",
```

Comment:
This will effectively result in prettier being run twice in each package, won't it?

Reply:
Hmmm, yes. I was hoping to avoid the prettier:root business in this release group but it looks necessary. I think the
long-term solution is to move these checks to be done separately from linting.